lib/parsers/mp3_parser/id3_extraction.rb

```diff
@@ -1,6 +1,9 @@
 module FormatParser::MP3Parser::ID3Extraction
   ID3V1_TAG_SIZE_BYTES = 128
-  ID3V2_TAG_VERSIONS = ["\x43\x00".b, "\x03\x00".b, "\x02\x00".b]
```

Comment:
You could've added a comment explaining what these versions are; though I don't mind this change either.

Reply:
yeah, but I think it's a bit easier to understand!